app/src/main/java/com/example/parkingsystem/mvp/view/base/FragmentView.java

```diff
@@ -0,0 +1,24 @@
+package com.example.parkingsystem.mvp.view.base;
+
+import android.content.Context;
+import androidx.annotation.Nullable;
+import com.example.parkingsystem.fragment.SpacesParkingDialogFragment;
+import java.lang.ref.WeakReference;
+
+public class FragmentView {
+    private WeakReference<SpacesParkingDialogFragment> fragmentRef;
```

Comment:
this should be a more general class like 
`    private WeakReference<Fragment> fragmentRef;`

Because if you let it like this it will only works for SpacesParkingDialogFragment and not for any fragment

Reply:
fixed